src/lib/page-state/_core.scss

```diff
@@ -0,0 +1,84 @@
+@use './token-utils' as *;
+@use '../core/styles/typography';
+
+@forward './token-utils';
+
+@mixin host {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  container-type: inline-size;
+}
+
+@mixin host-hidden {
+  display: none;
+}
+
+@mixin base {
+  width: #{token(width)};
+  margin-top: #{token(spacing)};
+  margin-bottom: #{token(spacing)};
+
+  text-align: center;
+}
+
+@mixin base-mobile {
+  @include override(width, mobile-width);
+}
+
+@mixin graphic {
```

Comment:
Can I suggest we rename the `graphic` mixin to `graphic-container` and rename `graphic-slotted` to `graphic`? I think this would better reflect what these mixins are intended to be applied to. The class names in the template are fine as-is IMO.